packages/core/src/lib/modal.svelte

```diff
@@ -26,81 +26,91 @@ Creates a modal overlay.
 
 <script lang="ts">
 import cx from 'classnames';
-import { createEventDispatcher, onMount } from 'svelte';
 import IconButton from './button/icon-button.svelte';
+import { clickOutside } from '$lib';
+import { writable } from 'svelte/store';
+import { onMount } from 'svelte';
 
 /** Whether the modal is open. */
-export let open = false;
+export let isOpen = writable(false);
```

Comment:
I don't think this should have a default value, because the component really isn't usable if you don't pass something in

```suggestion
export let isOpen: Writable<boolean>;
```